### DISCUSSION TITLE
part-ii/2021-100/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#158] [Question comments]
Part II 2021 Number Fields: Paper 1, Section II, 20G 

https://questions.tripos.org/part-ii/2021-100/

**Comment by dropdbassett** [comment]
<details><summary>Part (a)</summary>

It suffices to show $T^3-5T+8$ is irreducible over $\mathbb{Q}$. Since it is cubic, we need only check for roots in $\mathbb{Q}$, and by the rational root theorem they must also be an integer dividing 8. It is possible to check all 8 possibilities; alternatively, if $\beta$ is a solution, note that $\beta-1$ is a root of 

$$(T+1)^3 - 5(T+1) + 8 \equiv T^3 + 3T^2 - 2T + 4,$$

and hence is an integer dividing 4. This limits the check to $\beta = 2$ or $\beta = -1$, and we have $2^3 - 5\cdot 2 + 8 = 6$, $(-1)^3 - 5 \cdot (-1) + 8 = -2$.
</details>

<details><summary>Part (b)</summary>

The matrix for $\beta$ acting by multiplication is given by

$$M_\beta = \begin{pmatrix}0 & -4 & -4\\\frac{1}{2} & \frac{5}{2} & -\frac{3}{2}\\\frac{1}{2} & \frac{1}{2} & \frac{5}{2}\end{pmatrix},$$

which has characteristic polynomial $-T^3 + 5T^2 - 11T + 12 \in \mathbb{Z}[T]$, so $\beta$ is an algebraic integer.

Since $\alpha$ is a primitive element for the field extension, $\mathrm{disc}(1,\alpha,\alpha^2) = \mathrm{disc}(T^3-5T + 8) = -4\cdot 307$. But $\lvert \mathbb{Z}[\alpha, \beta] : \mathbb{Z}[\alpha]\rvert = 2$, and hence $\mathrm{disc}(1, \alpha, \beta) = -307$ is square-free. Thus $(1,\alpha, \beta)$ is a $\mathbb{Z}$-basis for $\mathcal{O}_K$.
</details>

<details><summary>Compute the prime factorisation of the ideal (3)</summary>

Since $\lvert\mathcal{O}_K : \mathbb{Z}[\alpha]\rvert = 2$ is coprime to 3, we may apply Dedekind to $\alpha$. But 

$$T^3 - 5T + 8 \equiv (T-2)(T^2-T-1) \pmod 3$$

giving $(3) = (3, \alpha - 1)(3, \alpha^2 - \alpha - 1)$.
</details>
<details><summary>Is (2) a prime ideal?</summary>

From the minimal polynomial of $\alpha$, we have 

$$\alpha(5-\alpha^2) = 8 \in (2).$$

But the elements of (2) have even constant coefficients, and the coefficients of $\alpha$ and $\alpha^2$ must have the same parity, since $1, \alpha, \beta$ is a $\mathbb{Z}$-basis. So (2) is not prime.
</details>

**Reply by artie2000** [reply]
I think this should be $T^3-5T+8=(T-2)(T^2-T-1)$

**Reply by dropdbassett** [reply]
You're right, thanks! (I transcribed wrongly the +1 as a -1, which is of course the same as what you have)

**Reply by Kiolt** [reply]
For the last part you can also show that $\mathcal{O}_K=\mathbb{Z}[\beta]$ and use Dedekind. Another way it to find an element of norm 6 which you can get by computing norms of $\alpha+n$ and remembering that $N_{K/\mathbb{Q}}(n+\alpha)=(-1)^{\mathrm{deg}_\mathbb{Q}\alpha}f(-n)$.
